apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,55 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemRefDirective } from '../../ui/list-item/ListItemTemplateRef.directive';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities()"
+      [imageUrl]="cityImage"
+      (addNewItem)="addNewCity()"
+      customClass="bg-light-blue">
+      <ng-template listItemRef let-city>
+        <app-list-item (delete)="deleteCity(city.id)">
+          {{ city.name }}
+        </app-list-item>
+      </ng-template>
+    </app-card>
+  `,
   standalone: true,
-  imports: [],
+  styles: [
+    `
+      ::ng-deep .bg-light-blue {
```

Comment:
> CSS: try to avoid using ::ng-deep. Find a better way to handle CSS styling.

I guess your code will work even without the `::ng-deep`.

Reply:
Updated css